go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/diagridio/go-etcd-cron
 go 1.22.5
 
 require (
-	github.com/dapr/kit v0.13.1-0.20240724000121-26b564d9d0f5
+	github.com/dapr/kit v0.13.1-0.20240909215017-3823663aa4bb
```

Comment:
I think we might be able to close this PR and let the List API PR supersede this one?

Reply:
:+1: